src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,166 @@
+import React, { useState } from 'react';
+
+import useData from 'hooks/useData';
+import useMediaQuery from 'hooks/useMediaQuery';
+
+import JoinFull from '@mui/icons-material/JoinFull';
+import PlayCircle from '@mui/icons-material/PlayCircle';
+import Storage from '@mui/icons-material/Storage';
+import TableChart from '@mui/icons-material/TableChart';
+import { Typography } from '@mui/material';
+import MuiAlert from '@mui/material/Alert';
+import AppBar from '@mui/material/AppBar';
+import Backdrop from '@mui/material/Backdrop';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Chip from '@mui/material/Chip';
+import Drawer from '@mui/material/Drawer';
+import Grid from '@mui/material/Grid';
+import IconButton from '@mui/material/IconButton';
+import Paper from '@mui/material/Paper';
+import Snackbar from '@mui/material/Snackbar';
+import Stack from '@mui/material/Stack';
+import { styled } from '@mui/material/styles';
+import Toolbar from '@mui/material/Toolbar';
+
+import { AVAILABLE_TABLES, TABLE_FIELDS } from '../../constants/constants';
+
+const Alert = React.forwardRef(function Alert(props, ref) {
+	return <MuiAlert elevation={6} ref={ref} variant="filled" {...props} />;
+});
+
+const Item = styled(Paper)(({ theme }) => ({
+	backgroundColor: theme.palette.mode === 'dark' ? '#1A2027' : '#fff',
+	...theme.typography.body2,
+	padding: theme.spacing(1),
+	textAlign: 'center',
+	color: theme.palette.text.secondary,
+}));
+
+function Navbar({ query, setQuery, value, setValue, history, setHistory }) {
+	const { showToast, toastMsg } = useData(query);
+	const [open, setOpen] = React.useState(false);
+	const [state, setState] = useState({
+		right: false,
+	});
+	const [backdropToggle, setBackdropToggle] = useState(false);
+	const [selectedTable, setSelectedTable] = useState('categories');
+	const isMobileView = useMediaQuery('(max-width: 700px)');
+
+	const toggleDrawer = (anchor, open) => (event) => {
+		if (event.type === 'keydown' && (event.key === 'Tab' || event.key === 'Shift')) {
+			return;
+		}
+		setState({ ...state, [anchor]: open });
+	};
+
+	const onSubmit = () => {
+		var Z = value.toLowerCase().slice(value.indexOf('from') + 'from'.length);
```

Comment:
Unless there's some convention you're following for this capital Z variable name, generally speaking we've been using camelCase and it'd be best if we stick to one convention. I'm also in the camp of descriptive variable names over short ones so it would be nice if the name of the variable could tell us what exactly its storing, I don't think its intuitive from looking at the rest of the line.